lib/src/excel.dart

```diff
@@ -164,46 +185,49 @@ class ExcelDart {
       sheet.activate();
 
       // All operations are complete.
-      populateCompleter.complete();
-
       return context.sync();
+    })).then(allowInterop((value) {
+      populateCompleter.complete();
     }));
 
     // Waits for population to be complete.
     await populateCompleter.future;
 
     // Conditionally format the budget column once the table is set.
-    await _formatBudgetColumn(tableName);
+    await _formatBudgetAndSpentColumns(tableName);
 
     // Highlight underpacing insertion orders if highlightUnderpacing is true.
-    if (highlightUnderpacing) await _highlightUnderpacingRows(tableName);
```

Comment:
Always apply the conditional formatting formula since I moved the condition to inside the formula ([see here](https://github.com/googleinterns/dv360-excel-plugin/pull/67/commits/9d4a03356e5c76bf6d3aae5bb8ed06ad7e6706ff#diff-5a9f11b722aad015b548cb227e5be190R250-R251))